grid/app/main/dfl/auth/session_repository.py

```diff
@@ -0,0 +1,70 @@
+from syft.grid.authentication.account import AccountCredential
+from .user_session import UserSession
```

Comment:
Imports should be grouped in the following order:

1.Standard library imports.
2.Related third party imports.
3.Local application/library specific imports.

You should put a blank line between each group of imports.